### PR TITLE
add autots use case

### PR DESCRIPTION
**Pull request by yushan111**
main changes:

- add auto ts use case

- change the name of model use case from "network_traffic_forecasting" to "network_traffic_model_forecasting"

-  fix bug of fill missing data in model use case

### CODE REVIEW

**Comment by shane-huang** [comment]
The flow generally looks good to me. Add a README in network_traffic folder describing scenario and the files.

**Comment by shane-huang** [comment]
Got a little bit concern that only 4 trials won't be enough to showcase the capability of autots.. .

File: pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb [hunk 1]
```diff
@@ -0,0 +1,1768 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Network Traffic Forecasting with _autoTS_"
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb]: AutoTS

File: pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb [hunk 2]
```diff
@@ -0,0 +1,1768 @@
...
+    "The general workflow using automated training contains below two steps. \n",
+    "   1. create a ```AutoTSTrainer``` to train a ```TSPipeline```, save it to file to use later or elsewhere if you wish.\n",
+    "   2. use ```TSPipeline``` to do prediction, evaluation, and incremental fitting as well. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, you need to initialize RayOnSpark before using auto training (i.e. ```AutoTSTrainer```), and stop it after training finished. Using TSPipeline only does not need RayOnSpark. \n"
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb]: (Note RayOnSpark is not needed if you just use TSPipeline for inference).

File: pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb [hunk 3]
```diff
@@ -0,0 +1,1768 @@
...
+      "CPU times: user 14.6 s, sys: 4.26 s, total: 18.9 s\n",
+      "Wall time: 8min 35s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "from zoo.zouwu.autots.forecast import GridRandomRecipe\n",
+    "ts_pipeline = trainer.fit(train_df, val_df, \n",
+    "                          recipe=GridRandomRecipe(look_back=look_back),\n",
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb]: need to briefly explain GridRandomRecipe

Comment (shane-huang) [reply pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb]: Actually we have a doc in github.io. Maybe we can add a link to the docs?

File: pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb (hunk 2 above)
Comment (yushan111) [reply pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb]: @jason-dai  `TSPipeline` can be also used for incremental fitting. How about we change to "(Note RayOnSpark is not needed if you just use TSPipeline)" ?

Comment (jason-dai) [reply pyzoo/zoo/zouwu/use-case/network_traffic/network_traffic_autots_forecasting.ipynb]: Note RayOnSpark is not needed if you just use TSPipeline for inference, evaluation or incremental training

**Comment by shane-huang** [comment]
Will change the interface to allow both simple arguments and Recipe.  Merge this PR for now and open a new one to do it.

**Pull request merged**